backend/src/xfd_django/xfd_api/tasks/refresh_vs_summaries.py

```diff
@@ -114,7 +114,10 @@ def handler(event):
 
         try:
             LOGGER.info("Creating Port summaries.")
-            create_port_scan_summary()
+            orgs = Organization.objects.all()
```

Comment:
Consider paginating this for stability, having everything in memory at once I would think at some point will cause an issue.